content/en/infrastructure/process/generate_process_metrics.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Generate Metrics from Processes
+title: Increase Process Retention
 kind: documentation
 aliases:
     - /processes/processing/process_to_metrics/
-    - /infrastructure/generate_process_metrics/
+    - /infrastructure/increase_process_retention/
```

Comment:
This is referring to the URL/filename, which you've left unchanged, so it breaks the site. Either we need to change it back or completely rename the file and setup a redirect from the old one to here.

Reply:
Got it, thanks! I'll change it back for now, and would you be able to kick off the process to rename it for the future?

Reply:
I will also follow up with the name change in the sidebar navigation at the same time.

Reply:
Thanks! Otherwise - is this good to go? (can I merge?)